test.js

```diff
@@ -88,38 +84,6 @@ test('Should register the hooks and trace the request (404)', t => {
   })
 })
 
-test('Should handle querystrings', t => {
-  const fastify = Fastify()
-
-  const record = sinon.spy()
-  const recorder = { record }
-  const ctxImpl = new ExplicitContext()
-  const serviceName = 'test'
-  const tracer = new Tracer({ recorder, ctxImpl })
-
-  ctxImpl.scoped(() => {
-    fastify.register(zipkinPlugin, { tracer, serviceName })
-
-    fastify.get('/', (req, reply) => {
-      reply.send({ hello: 'world' })
-    })
-
-    fastify.inject({
-      url: '/?foo=bar',
-      method: 'GET'
-    }, (err, res) => {
-      t.error(err)
-
-      const annotations = record.args.map((args) => args[0])
-      t.strictEqual(annotations[2].annotation.annotationType, 'BinaryAnnotation')
-      t.strictEqual(annotations[2].annotation.key, 'http.url')
-      t.strictEqual(annotations[2].annotation.value, '/?foo=bar')
-
-      t.end()
-    })
-  })
-})
```

Comment:
Why was this test removed?

Reply:
Removed that test, because, for security reasons, Zipkin has removed support for http.url and instead are only using http_path property, which does not contain the query parameters, and so that test becomes irrelevant with the new version of Zipkin.

Reply:
https://github.com/openzipkin/zipkin-js/pull/226